model/event.go

```diff
@@ -362,6 +373,41 @@ WHERE
 	return event.ID, nil
 }
 
+func UpdateEventSurveyStatus(db *sqlx.DB, event Event) (eventID int, err error) {
+	tx, err := db.Beginx()
+	if err != nil {
+		return 0, errors.Wrap(err, "failed to update event")
+	}
+	// Error out if the event doesn't exist.
+	var eventCount int
+	err = tx.Get(&eventCount, `SELECT count(*) FROM events WHERE id = ?`, event.ID)
```

Comment:
Doesn't `UPDATE` return the number of rows affected? Seems like having an extra query like this is unnecessary.

Reply:
Yeah, it does seem unnecessary for this. I had just borrowed it from updateEvent().